src/App.js

```diff
@@ -1,43 +1,55 @@
 import Board from "./components/Board";
 import Hex from "./components/Hex";
 import Switch from "./components/Switch";
-import { useState } from 'react';
+import Clear from "./components/Clear";
+import History from "./components/History";
+import { useState, useEffect } from 'react';
 
 
 function App() {
 
-  const [ color, setColor] = useState('#ff3362');
+
+  const [ history, setHistory] = useState(JSON.parse(localStorage.getItem('colorHistory')) || ['#ff3362']);
+  const [ color, setColor] = useState(history[0] || '#ff3362');
+  
   
   const changeColor = ()=>{
 
-    let hx = (Math.random() * 0xfffff * 100000).toString(16).slice(0,6);
-    setColor(
-      `#${hx}`
-    )
+    let currentColor = `#${(Math.random() * 0xfffff * 100000).toString(16).slice(0,6)}`;
+    setColor(currentColor);
+
+    setHistory([currentColor, ...history].slice(0,6));
```

Comment:
@Richd0tcom Something here makes it keep only 5 colors in history rather than the recommended 10.

Reply:
Yeah I initially made it 5 to test it. I make the corrections as soon as possible